src/components/QuicksightDashboardPage.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import { Store } from "redux";
+import { RootState } from "../store";
+import * as PropTypes from "prop-types";
+import Header from "./Header";
+import Footer from "./Footer";
+import title from "../utils/title";
+import QuicksightDashboard from "./QuicksightDashboard";
+
+export interface QuicksightDashboardPageProps
+  extends React.Props<QuicksightDashboardPageProps> {
+  params: {
+    library?: string;
+  };
+}
```

Comment:
Since we're no longer passing a `library` param here, we no longer need the `QuicksightDashboardPageProps` interface, so we should get rid of this.

We'll also need to remove it from the class definition below.

Reply:
We're not passing it now, but we may want to pass it later.  Ie if we want to pre-select the library.  To make this work, we'll need to make some changes on the backend.  But I'm thinking we may want to leave this param in here for the time being.